### PR TITLE
Add Typescript typings

### LINKED ISSUE #8: Using with typescript problem
The repo is missing a typescript typings file.  Please add.

### PR DESCRIPTION
**Pull request by aalmazan**
## Description of the change

Adds Typescript typings for packages

image

image

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

Closes #8 

## Dev Checklist

- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development
- [ ] No previous tests unrelated to the changed code fail in development

### CODE REVIEW

File: packages/react-signature-frame/package.json [hunk 1]
```diff
@@ -5,12 +5,14 @@
...
   "main": "dist/index.js",
+  "types": "./types/index.d.ts",
   "scripts": {
     "build": "webpack --mode production",
     "build:changelog": "yarn auto-changelog",
     "pack": "yarn pack",
     "version": "yarn build:changelog && git add CHANGELOG.md",
-    "test": "yarn mocha --config ./test/mocha.js"
+    "test": "yarn mocha --config ./test/mocha.js",
+    "tsc": "tsc"
```
Comment (newhouse) [inline packages/react-signature-frame/package.json:15]: This necessary?

File: packages/react-signature-frame/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (newhouse) [inline packages/react-signature-frame/tsconfig.json:1]: Assuming this looks a lot like the other one from the `node-anvil`?

**Review by newhouse — APPROVED** [review]

File: packages/react-signature-frame/package.json (hunk 1 above)
Comment (aalmazan) [reply packages/react-signature-frame/package.json:15]: Yeah suppose not. I actually need to add that into the build script instead.

**Comment by benogle** [comment]
👏

**Pull request merged**